src/Managers/FolderManager.php

```diff
@@ -29,10 +29,12 @@ class FolderManager extends AbstractManager implements FolderManagerInterface, L
      *
      * @param int $page
      * @param int $perPage
+     * @param array $filters

```

Comment:
According to the interface it should be `array|null `

Reply:
No right? Because you cannot literally pass `null`?

Reply:
It has a default value, but not a type on the method itself. 
So if you ignore the docblock you can pass in literally anything 😉